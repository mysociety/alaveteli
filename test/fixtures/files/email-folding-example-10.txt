Please note: it is not possible to reply to this email. To contact the
Department of Health, please visit the 'Contact us' page on the
Department’s website.

-----------------------------------------------------------------------------------------
                                                                            
 Apologies that you were not able to read our previous response of 4        
 October. Please find the text of that email below.                         
                                                                            
Our ref: DE00000642471

Dear Ms Peters Rock, 

You requested your correspondence to be treated under the Freedom of
Information Act.  However, as your correspondence asked for general
information, rather than requesting recorded information or documentation,
I should advise you that on this occasion we have not considered your
correspondence under the provisions of the Act.

I am sorry I cannot be more helpful.

Yours sincerely, 
Simon Dove
Customer Service Centre
Department of Health




-------------------------------------------------------------------------------------------------------------------------


Please do not reply to this email. To contact the Department of Health,
please visit the 'Contact us' page on the Department’s website, where you
can also view our performance against quarterly service targets.


- - Disclaimer - -
This e-mail and any files transmitted with it are confidential. If you are
not the intended recipient, any reading, printing, storage, disclosure,
copying or any other action taken in respect of this e-mail is prohibited
and may be unlawful. If you are not the intended recipient, please notify
the sender immediately by using the reply function and then permanently
delete what you have received.

Incoming and outgoing e-mail messages are routinely monitored for
compliance with the Department of Health's policy on the use of electronic
communications. For more information on the Department of Health's e-mail
policy click here http://www.dh.gov.uk/terms

The original of this email was scanned for viruses by the Government Secure Intranet virus scanning service supplied by Cable&Wireless Worldwide in partnership with MessageLabs. (CCTM Certificate Number 2009/09/0052.) On leaving the GSi this email was certified virus free.
Communications via the GSi may be automatically logged, monitored and/or recorded for legal purposes.
