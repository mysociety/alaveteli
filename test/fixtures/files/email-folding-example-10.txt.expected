Please note: it is not possible to reply to this email. To contact the
Department of Health, please visit the 'Contact us' page on the
Department’s website.

-----------------------------------------------------------------------------------------
                                                                            
 Apologies that you were not able to read our previous response of 4        
 October. Please find the text of that email below.                         
                                                                            
Our ref: DE00000642471

Dear Ms Peters Rock, 

You requested your correspondence to be treated under the Freedom of
Information Act.  However, as your correspondence asked for general
information, rather than requesting recorded information or documentation,
I should advise you that on this occasion we have not considered your
correspondence under the provisions of the Act.

I am sorry I cannot be more helpful.

Yours sincerely, 
Simon Dove
Customer Service Centre
Department of Health
